digitaltwin_client/samples/digitaltwin_sample_environmental_sensor/digitaltwin_sample_environmental_sensor.c

```diff
@@ -603,25 +608,37 @@ DIGITALTWIN_CLIENT_RESULT DigitalTwinSampleEnvironmentalSensor_SendTelemetryMess
 
     float currentTemperature = 20.0f + ((float)rand() / RAND_MAX) * 15.0f;
     float currentHumidity = 60.0f + ((float)rand() / RAND_MAX) * 20.0f;
-
+    float currentLongitude = -124.0f + ((float)rand() / RAND_MAX) * 59.0f;
```

Comment:
>59.0f [](start = 68, length = 5)

Recommend we make this and the 25 on latitude multiplier much smaller.  If someone showed this on a map - and folks do that - it would be jumping all over big part of world every 20 seconds.

Reply:
The coordinate set used here is within USA.  I will change it to approximately within Washington state.